python_modules/libraries/dagster-omni/dagster_omni/objects.py

```diff
@@ -82,6 +84,8 @@ def from_json(cls, data: dict[str, Any], queries: list["OmniQuery"]) -> "OmniDoc
             folder=folder,
             labels=labels,
             queries=queries,
+            favorites=data.get("_count", {}).get("favorites", None),
+            views=data.get("_count", {}).get("views", None),
```

Comment:
wonder if we should include an updated timestamp in the metadata too so end users get a sense of how outdated these #s might be

Reply:
we have information about the age of the state in the the actual DefsStateStorage, so I think that'd be the nice generic way of giving people insight into this (just would require more UI piping)